### ISSUE TITLE
Dimple 使用的数据

### ISSUE THREAD

**Issue by lsm2842035890** [yu-rp/Dimple#4]
https://huggingface.co/datasets/liuhaotian/LLaVA-CC3M-Pretrain-595K/tree/main
论文里说的数据集是来自这个链接里的吗~
Phase I: Autoregressive Training In the first phase, we modify the attention mechanism within
Dream by replacing the full attention mask with a causal attention mask, enabling the model to
operate under an autoregressive decoding paradigm. **This phase is designed for vision-language
alignment and then visual instruction following learning**. We follow the LLaVA-NEXT [22] training
recipe and use its datasets.
Phase II: Diffusion Training In the second phase, we transition back to the diffusion generation
framework. To this end, we re-enable the full attention mask within Dream. **The LLaVA-NEXT
instruction-following data is used again**. The loss in eq. (3) is used. The mask is only applied to the
answer part of each sample.

**Comment by yu-rp** [comment]
@lsm2842035890 ,

感谢 Issue 。 第一阶段的 alignment 数据 是的。具体的数据如下：

Phase 1 - AR alignment: https://huggingface.co/datasets/liuhaotian/LLaVA-CC3M-Pretrain-595K/tree/main
Phase 1 - AR instruction tuning: https://huggingface.co/datasets/lmms-lab/LLaVA-NeXT-Data

Phase 2 - Diff instruction tuning: https://huggingface.co/datasets/lmms-lab/LLaVA-NeXT-Data

**Comment by lsm2842035890** [comment]
https://huggingface.co/datasets/lmms-lab/LLaVA-NeXT-Data 这个数据集很大呀，我们的服务器不能访问外网，下载下来要很久，我看你的代码里是直接访问huggface加载，速度怎么样～

**Comment by yu-rp** [comment]
直接从huggingface 上下载可能比较方便。如果没法直接访问，那 clone 整个仓库可以么？
